pkgs/top-level/config.nix

```
@@ -113,6 +113,23 @@ let
       '';
     };
 
+    allowUnfreePackages = lib.mkOption {
+      type = with lib.types; listOf str;
+      default = [ ];
+      example = [ "ut1999" ];
+      description = ''
+        Allows specific unfree packages to be used.
+
+        This option overrides `nixpkgs.config.allowUnfreePredicate`with a function that permits the listed package names.
```

Add missing blank space

```suggestion
        This option overrides `nixpkgs.config.allowUnfreePredicate` with a function that permits the listed package names.
```